.github/actions/setup-zeebe/action.yml

```diff
@@ -31,6 +31,10 @@ inputs:
   secret_vault_secretId:
     description: 'secret valut secret id'
     required: false
+  docker:
```

Comment:
:thought_balloon:  rather than adding yet another input variable, we could set `continue-on-error: true`. It's a bit disappointing that `docker/login-action` does not support windows and mac :disappointed:

Reply:
Good point, I didn't think about that :+1: